server/subscription.go

```diff
@@ -68,11 +60,11 @@ func (server *Server) loadSubscription() *enterprise.Subscription {
 			ExpiresTs:     license.ExpiresTs,
 			InstanceCount: license.InstanceCount,
 		}
-	} else {
-		// increase instance quota for dev environment.
-		if server.mode == "dev" {
-			subscription.InstanceCount = subscription.InstanceCount * 2
-		}
+	}
+
+	// change instance quota for dev environment.
```

Comment:
Maybe we don't need this? Just let client call patchSubscription with dev license upon startup?